Code/GraphMol/MolDraw2D/MolDraw2D.cpp

```diff
@@ -1068,6 +1069,10 @@ pair<string, MolDraw2D::OrientType> MolDraw2D::getAtomSymbolAndOrientation(
     symbol = "";
   } else if (isComplexQuery(&atom)) {
     symbol = "?";
+  } else if (drawOptions().atomLabelDeuteriumTritium && atom.getAtomicNum() == 1 &&
```

Comment:
When I played with some of the query stuff, I found it quite hard to make free functions with appropriate business logic or state.  It seems like a functor interface (callback-like) can have the best of both worlds.  Functor interfaces can have the best of both worlds and in my experience are easier to wrap in Python/SWIG.  Or we could just go to c++11 :)